source/stringified.d.ts

```diff
@@ -0,0 +1,20 @@
+/**
+Create a type from an object type without certain keys.
+
+@example
+```
+import {Stringified} from 'type-fest';
+
+type Foo = {
+	a: number;
+	b: string;
+};
+
+type FooWithoutA = Stringified<Foo, 'a'>;
```

Comment:
`Stringified` only has one generic argument: `ObjectType`

What does `'a'` map to?

Reply:
Yes, I was wrong on that. I fixed it